src/transformers/file_utils.py

```diff
@@ -297,10 +297,10 @@ def is_torch_cuda_available():
         return False
 
 
-_torch_fx_available = False
+_torch_fx_available = _torch_onnx_dict_inputs_support_available = False
 if _torch_available:
     torch_version = version.parse(importlib_metadata.version("torch"))
-    _torch_fx_available = (torch_version.major, torch_version.minor) == (
+    _torch_fx_available = _torch_onnx_dict_inputs_support_available = (torch_version.major, torch_version.minor) == (
         TORCH_FX_REQUIRED_VERSION.major,
         TORCH_FX_REQUIRED_VERSION.minor,
     )
```

Comment:
We should have a different check as torch_fx requires PyTorch 1.8 exactly and `_torch_onnx_dict_inputs_support_available` requires PyTorch 1.8 or greater.